include/seqan3/alignment/matrix/trace_directions.hpp

```diff
@@ -56,17 +56,37 @@ namespace seqan3
  * \param s The seqan3::debug_stream.
  * \param trace The trace direction.
  * \relates seqan3::debug_stream_type
+ *
+ * \details
+ *
+ * The following table shows the printed symbol of a particular seqan3::detail::trace_directions:
```

Comment:
This sentence highlights why we use generic singular. The enum should be called `seqan3::detail::trace_direction` but thats a different issue.

Reply:
I know, something that happens sometimes later together with a bunch of clean up steps